src/main/scala/com/fulcrumgenomics/illumina/SampleSheet.scala

```diff
@@ -149,6 +150,16 @@ object SampleSheet {
   }
 
   protected[illumina] val SplitRegex = ",(?=(?:[^\"]*\"[^\"]*\")*[^\"]*$)"
+
+  /** Validate that the sample ids are unique, and that the combination of sample name and library id are unique. */
+  private def validateSamples(samples: Seq[Sample]): Unit = {
+    // Validate some properties for a sample sheet
+    require(samples.map(_.sampleId).toSet.size == samples.length,
+      "Sample identifiers were not unique: " + samples.groupBy(_.sampleId).filter(_._2.length > 2).keys.mkString(", "))
+    require(samples.map(s => s"${s.sampleName} ${s.sampleName}").toSet.size == samples.length,
```

Comment:
I don't understand why this is `s.sampleName` twice - I think that's probably a mistake?  Should the second value on this line and on 159 be `s.libraryId`?